### PR TITLE
Multi-language support

### PR DESCRIPTION
**Pull request by domingoruiz**
Hi. I have added multi-language support again. Now I have also added a menu to select the language to use in the footer.
Bye

### CODE REVIEW

**Comment by FoggyMtnDrifter** [comment]
Looks great! Only thing is we need the spelling to be "language" and not "languaje". Can you correct this and we can get @hbjydev to review and merge it?

Also, I think we decided we would work on the drop-down later, but we can get @hbjydev's input on that.

**Comment by FoggyMtnDrifter** [comment]
Also, it looks like something is messed up with the footer, it ends abruptly towards the bottom. We will need that corrected, as well before we consider merging.

**Comment by hbjydev** [comment]
@ressonix is correct, we said we'll handle the language selector, so you can remove that part. Just leave links or remove the selector altogether -- this'll be merged into `develop` first before progressing to `main`.

Also, the changes I wanted made were that there are some typos in language names & the wrong string used in the FAQ. I'll submit a review for those.

**Comment by domingoruiz** [comment]
Hi. I have solved the bug when writing the word language in all places. In addition, I have improved the display of the menu a bit, and I have included a javascript that reads the browser language and selects the site language. You are already telling me.

Bye.

**Comment by hbjydev** [comment]
@domingoruiz As we already said, let us handle the menu ourselves. I've remove that from your PR.

On top of this, can you update the `es.toml` file to match the layout of the `en.toml` file?

File: i18n/es.toml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+other = "El proyecto CentOS anunció recientemente un cambio en la estrategia de CentOS. Mientras que anteriormente CentOS existía como una compilación descendente de su proveedor ascendente (recibe parches y actualizaciones después de que lo hace el proveedor ascendente), se cambiará a una compilación ascendente (probando parches y actualizaciones antes de su inclusión en el proveedor ascendente). Además, el soporte para CentOS Linux 8 se ha interrumpido, desde el 31 de mayo de 2029 hasta el 31 de diciembre de 2021."
+
+[rocky-inux-come-in]
+other = "Entonces, ¿dónde entra Rocky Linux?"
+
+[rocky-inux-come-in-solve]
+other = "Rocky Linux apunta a funcionar como una compilación descendente como lo había hecho CentOS anteriormente, construyendo versiones después de que hayan sido agregadas por el proveedor ascendente, no antes."
+
+[select-language]
+other = "Seleccionar idioma"
```
Comment (hbjydev) [inline i18n/es.toml]: Please update this format to match that of `en.toml`

**Review by hbjydev — CHANGES_REQUESTED** [review]

File: i18n/es.toml (hunk 1 above)
Comment (domingoruiz) [reply i18n/es.toml]: Ok, it's solved.

**Review by hbjydev — APPROVED** [review]

**Pull request merged**